packages/xo-web/src/common/tags.js

```diff
@@ -85,6 +108,26 @@ export default class Tags extends Component {
     event.preventDefault()
   }
 
+  _selectExistingTags = () =>
+    confirm({
+      body: <SelectExistingTag />,
+      icon: 'add',
+      title: _('selectExistingTags'),
+    })
+      .then(tags => Promise.all(tags.map(this._addTag)))
+      ::pFinally(this._stopEdit)
```

Comment:
You'd need to put this first otherwise it will still wait for the tags to be added before closing the edition.